Augment the list of expected symbols to fix the symbol-check tests on
mips64el-linux, armhf-linux and aarch64-linux.

--- libdrm-2.4.65/freedreno/freedreno-symbol-check.orig	2015-09-04 11:07:40.000000000 -0400
+++ libdrm-2.4.65/freedreno/freedreno-symbol-check	2015-10-18 23:57:15.288416229 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.sources/LIBDRM_FREEDRENO_H_FILES
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_freedreno.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 fd_bo_cpu_fini
 fd_bo_cpu_prep
 fd_bo_del
--- libdrm-2.4.65/nouveau/nouveau-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/nouveau/nouveau-symbol-check	2015-10-18 23:55:26.078327118 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.sources/LIBDRM_NOUVEAU_H_FILES
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_nouveau.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 nouveau_bo_map
 nouveau_bo_name_get
 nouveau_bo_name_ref
--- libdrm-2.4.65/libkms/kms-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/libkms/kms-symbol-check	2015-10-18 23:46:10.683869471 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.sources/LIBKMS_H_FILES
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libkms.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 kms_bo_create
 kms_bo_destroy
 kms_bo_get_prop
--- libdrm-2.4.65/intel/intel-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/intel/intel-symbol-check	2015-10-18 23:55:53.309558508 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.sources/LIBDRM_INTEL_H_FILES
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_intel.so} | awk '{print $3}' | while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 drm_intel_bo_alloc
 drm_intel_bo_alloc_for_render
 drm_intel_bo_alloc_tiled
--- libdrm-2.4.65/amdgpu/amdgpu-symbol-check.orig	2015-08-17 10:08:11.000000000 -0400
+++ libdrm-2.4.65/amdgpu/amdgpu-symbol-check	2015-10-18 23:56:10.606917723 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.am/libdrm_amdgpuinclude_HEADERS
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_amdgpu.so} | awk '{print $3}' | while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 amdgpu_bo_alloc
 amdgpu_bo_cpu_map
 amdgpu_bo_cpu_unmap
--- libdrm-2.4.65/exynos/exynos-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/exynos/exynos-symbol-check	2015-10-18 23:56:32.025486153 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.am/libdrm_exynos*_HEADERS
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_exynos.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 exynos_bo_create
 exynos_bo_destroy
 exynos_bo_from_name
--- libdrm-2.4.65/omap/omap-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/omap/omap-symbol-check	2015-10-18 23:56:44.834438626 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.am/libdrm_omap*HEADERS
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_omap.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 omap_bo_cpu_fini
 omap_bo_cpu_prep
 omap_bo_del
--- libdrm-2.4.65/tegra/tegra-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/tegra/tegra-symbol-check	2015-10-18 23:57:00.756759698 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first nine) are taken from tegra.h.
+# The following symbols (past the first 12) are taken from tegra.h.
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_tegra.so} | awk '{print $3}'| while read func; do
 ( grep -q "^$func$" || echo $func )  <<EOF
@@ -9,6 +9,9 @@
 __bss_start
 __end__
 _bss_end__
+_fbss
+_fdata
+_ftext
 _edata
 _end
 _fini
--- libdrm-2.4.65/radeon/radeon-symbol-check.orig	2015-05-04 11:47:43.000000000 -0400
+++ libdrm-2.4.65/radeon/radeon-symbol-check	2015-10-18 23:57:00.756759698 -0400
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The following symbols (past the first five) are taken from the public headers.
+# The following symbols (past the first 12) are taken from the public headers.
 # A list of the latter should be available Makefile.sources/LIBDRM_RADEON_H_FILES
 
 FUNCS=$(nm -D --format=bsd --defined-only ${1-.libs/libdrm_tegra.so} | awk '{print $3}'| while read func; do
@@ -10,6 +10,13 @@
 _end
 _fini
 _init
+_fbss
+_fdata
+_ftext
+__bss_start__
+__bss_end__
+_bss_end__
+__end__
 radeon_bo_debug
 radeon_bo_get_handle
 radeon_bo_get_src_domain
